ReportSimulationOutput/measure.rb

```diff
@@ -812,11 +842,15 @@ def get_outputs(runner, args)
     end
 
     # Unmet Hours
-    @unmet_hours.each do |_key, unmet_hour|
+    @unmet_hours.each do |key, unmet_hour|
       unmet_hour.annual_output = get_report_variable_data_annual(['EMS'], ["#{unmet_hour.ems_variable}_annual_outvar"], 1.0)
       if args[:include_timeseries_unmet_hours]
         unmet_hour.timeseries_output = get_report_variable_data_timeseries(['EMS'], ["#{unmet_hour.ems_variable}_timeseries_outvar"], 1.0, 0, args[:timeseries_frequency])
       end
+
+      if key == UHT::Driving && unmet_hour.annual_output > 0.0
```

Comment:
Should we have some tolerance here? It doesn't seem particularly alarming if there's 1 hour of unmet driving, for example.

Reply:
As current users of this output in ResStock we find it useful to validate our inputs, it also indicates a lack of potential for peak shedding, even with just 1 hour showing up in the baseline.